src/org/labkey/test/pages/ReactAssayDesignerPage.java

```diff
@@ -53,12 +52,29 @@ public ReactAssayDesignerPage setName(String name)
         return this;
     }
 
+    public String getName()
+    {
+        return elementCache().nameInput.get();
+    }
+
+    // I think the name field is the only field that can be enabled/disabled.
+    // For example once an assay is created you can't change it's name so the field will be disabled.
+    public boolean isNameEnabled()
+    {
+        return null == elementCache().nameInput.getComponentElement().getAttribute( "disabled");
```

Comment:
If we're using the `disabled` attribute, I think the standard `isEnabled()` method would work. (Though perhaps not)
```suggestion
        return elementCache().nameInput.getComponentElement().isEnabled();
```

Reply:
Ok, we will give it a shot and see.